frontend/src/common/HomepageCard.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+HomepageCard.propTypes = {
+    children: PropTypes.any.isRequired,
+}
+
+export default function HomepageCard({ children }) {
+    return <CardBG>{children}</CardBG>
+}
+
+const CardBG = styled.div`
+  display: block;;
```

Comment:
```suggestion
  display: block;
```
meckert er bei zwei ;; ?